data/config.txt

```diff
@@ -0,0 +1,18 @@
+bye:bye
```

Comment:
Please try not to push your config file, as if I have different shortcut, it will be overwrite when i pull

Reply:
There is not config file in data/ of the team repository, should not we push at least one time ?

Reply:
Don't need, the way I have done the config file, if it doesn't exist initially, it will create one and fill it itself
But if you push your config file, and everytime it overwrite somebody else config file, it may be annoying

Reply:
ok